### ISSUE TITLE
mergedirs is not a function, contrary to all examples.

### ISSUE THREAD

**Issue by jsdw** [binocarlos/merge-dirs#5]
The example code looks like this:

```
var mergedirs = require('merge-dirs');

// copy folder/a into folder/b 
mergedirs('/folder/a', '/folder/b');
```

But actually the result of `require("merge-dirs")` is an object of `{conflictResolvers,default}` where `default` is the mergedirs function. I suspect it's a mismatch between using ES6 and the transpiled output

**Comment by binocarlos** [comment]
@jsdw Good point I see so it's the way an ES6 module is being loaded using `require`...

I think the solution would be to split out `conflictResolvers` into it's own module and have a single export inside of `src/index.js` - I've made a PR that does this: #6 

Could you check this for me and I will merge it - the tests pass - thanks!

**Comment by yesitsfebreeze** [comment]
you can work around this with 
```javascript
const mergedirs = require('merge-dirs').default;
```

**Comment by asifashraf** [comment]
Please update examples on npm page and github

**Comment by jpike88** [comment]
@binocarlos please can you bring this up to speed? it's just a merge and a publish

**Comment by jpike88** [comment]
@binocarlos are you maintaining this repo anymore? If not, can you hand it over to someone who will? The PR to fix the problem is simple, it would take you only a few minutes to merge and update this package. Please let us know.
